pkg/reconciler/revision/resources/deploy.go

```diff
@@ -262,13 +263,21 @@ func MakeDeployment(rev *v1.Revision, cfg *config.Config) (*appsv1.Deployment, e
 	}
 
 	replicaCount := cfg.Autoscaler.InitialScale
-	ann, found := rev.Annotations[autoscaling.InitialScaleAnnotationKey]
+	_, ann, found := autoscaling.InitialScaleAnnotation.Get(rev.Annotations)
```

Comment:
same question, this is in `autoscaling` but the annotation is `serving.knative.dev`

Reply:
All I did here was update to the new lookup method